ros2cli/package.xml

```diff
@@ -9,6 +9,7 @@
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>Apache License 2.0</license>
 
+  <exec_depend>python3-pkg-resources</exec_depend>
```

Comment:
Ok, I think that is fine for now with the `setuptools` dependency being injected by bloom.